MechJeb2/MechJebModuleStagingController.cs

```diff
@@ -206,7 +251,7 @@ public static bool InverseStageDecouplesActiveOrIdleEngineOrTank(int inverseStag
             {
                 Part p = v.parts[i];
                 Part decoupledPart;
-                if (p.inverseStage == inverseStage && p.IsUnfiredDecoupler(out decoupledPart) &&
+                if (p?.inverseStage == inverseStage && p.IsUnfiredDecoupler(out decoupledPart) &&
```

Comment:
I have some doubt about the use of this operator given how Unity override the null check but I guess that at worse it will behave as if it is not there.